en/payloads/README.md

```diff
@@ -0,0 +1,109 @@
+# Payloads and Cameras
+
+PX4 supports a wide range of payloads and cameras.
+
+## Mapping Drones
+
+* [Camera triggering](../peripherals/camera.md) via GPIO out
+* [Camera triggering](../peripherals/camera.md) via PWM out
+* [Camera triggering](../peripherals/camera.md) via MAVLink out
+* [Camera timing](../peripherals/camera.md#camera_capture) feedback via hotshoe input
+
+## Cargo drones and alike: Servos / Outputs
+
+* Servo or GPIO triggering (via RC or via commands)
```

Comment:
@bkueng For "via RC" is this the [RC_MAP_AUXx](http://docs.px4.io/master/en/advanced_config/parameter_reference.html#RC_MAP_AUX1) values?
My understanding is that you also have to have these AUXx values assigned in the mixer too? Right? So in practise this is a bit of a pain?

Reply:
Yes RC_MAP_AUX.
Yes, but most airframes have passthrough by default as well.

Reply:
Great. I've added a section on RC Trigger and merged the other PR. I never really got this before.